purchase.adoc

```diff
@@ -388,288 +517,533 @@ Fields named "*UUID1" are the standard UUID Type 1 (8-4-4-4-12) representation o
+If the purchase discount is 20%, then the price of each row of item will be deducted with 20% before calculating the VAT for each row.
 
-#### Response structure
-Row discounts are expressed using the following JSON structure in the `products` list, containing either `amount` for fixed discounts or `percentage` for percentage discounts. The `discountValue` field following the `discount` structure contains the total discount amount for the row:
-```json
+If a purchase has a total price of 100 and contains two rows with the first row priced at 40 and the second at 60, then 40% of the fixed purchase discount will be deducted from the first row and 60% from the other row before calculating the VAT.
+
+* Row discounts are expressed using the following JSON structure in the `products` list, containing either `amount` for fixed discounts or `percentage` for percentage discounts.
+The `discountValue` field following the `discount` structure contains the total discount amount for the row:
++
+--
+[source,json]
+----
 "discount": {
-  "quantity": 1,
-  "percentage": 10
+    "name": "10% off",
+    "quantity": 1,
+    "percentage": 10
 },
 "discountValue": 1000
-```
+----
+
 *Note:* Currently, we only support one discount per row of items, so the `quantity` field will always be set to 1. This could potentially change in the future.
+--
 
-Cart discounts are expressed using the following JSON structure in the root structure of the purchase. They can also contain either an `amount` or a `percentage` value:
-```json
+* Discounts for the whole purchase are expressed using the following JSON structure.
++
+--
+They can contain either `amount` or `percentage` value.
+
+[source,json]
+----
 "discounts": [
 {
-    "name": "",
     "amount": 2000,
     "quantity": 1
 }
-```
-*Note:* Currently, we only support one discount per cart, so the `discounts` list will only contain one item with `quantity` set to 1. This could potentially change in the future.
-
-See further below for a full response example containing discounts.
-
-#### Example
-Lets observe an example purchase, containing both discounts on rows as well as on the cart itself:
-[options="header"]
-|===
-| Product | Unit price | Amount | Discount
-| Foo | 20 | 2 | 5
-| Bar | 10 | 10 | 10%
-| Cart discount | | | 20%
-|===
-To calculate the total discount, we start by calculating the discount per row. The first row contains 2 item and has a fixed discount amount of 5.
-
-The second row contains 10 items and a percentage discount of 10%:
-
-*Row discounts* = 5 + 10 x 10 x 0.1 = 15.
-
-Then we calculate the cart discount, based on the sum of the previously discounted rows:
+----
 
-*Sum of discounted rows* = 20 x 2 + 10 x 10 - 15 = 125.
+*Note:* Currently, we only support one discount for the whole, so the `discounts` list will only contain one item with `quantity` set to 1. This could potentially change in the future.
```

Comment:
Usually the first person pronouns like we and I are not used in instructional tech docs, except in blog posts and marketing materials.